docs/installation/troubleshooting/pod_and_service_CIDR_conflicts.rst

```diff
@@ -0,0 +1,30 @@
+Pod and Service CIDR conflicts
+==============================
+
+If after installating a MetalK8s cluster you notice that the Pod-to-Pod
+communication has routing issues, perform the following checks:
+
+#. Check the configured values for the internal pod and service networks.
+
+   .. code-block:: shell
+
+      [root@bootstrap vagrant]# salt-call pillar.get networks
```

Comment:
would just remove vagrant as it is what you get when running on a dev environment but not on production environment